creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -562,14 +653,32 @@ class SnapbackSM {
        * Issue all required replica set updates
        * TODO move to chunked parallel (maybe?) + wrap each in try-catch to not halt on single error
        */
-      let numUpdateReplicaOpsIssued
+      let numUpdateReplicaOpsIssued = 0
+      let healthyNodes
       try {
         for await (const userInfo of requiredUpdateReplicaSetOps) {
+          // If `healthyNodes` is undefined, fetch all the healthy nodes, excluding the current replica set
+          // while disabling sync checks
+          if (!healthyNodes) {
```

Comment:
this flow doesn't realy make sense - for 10 requiredUpdateREeplicaSetOps, it would only make a call to to fetch the healhtyServicesMap the first time, and use the first user's replica set as blacklist. it would make more sense to just fetch it once outside the `for-await` with zero blacklist, since you will get the entire healthy serivces set anyway, and you can blacklist in memory afterward, per user

Reply:
discussed offline - will get healthy peers each modulo slice